GliaWidgets/SwiftUI/Extensions/View+Extensions.swift

```diff
@@ -103,4 +103,26 @@ extension SwiftUI.View {
             )
         }
     }
+
+    @ViewBuilder
+    func redactedPlaceholder(
+        _ colorType: ColorType,
+        font: UIFont? = nil,
+        cornerRadius: CGFloat = 2
+    ) -> some View {
+        let padding: CGFloat = if let font = font {
```

Comment:
Where `padding` is used?

Reply:
It seems on line 124

Reply:
Yes, it is in line 124.
It is because the placeholder is lower than the Text view height. Do make it similar to the original `.redacted(reason: .placeholder)` we should make it a bit lower.